test/distributed/_composable/fsdp/test_fully_shard_state.py

```diff
@@ -0,0 +1,93 @@
+# Owner(s): ["oncall: distributed"]
+
+import copy
+import unittest
+
+import torch
+import torch.nn as nn
+from _test_fully_shard_common import MLP
+
+from torch.distributed._composable.fsdp import FSDP, fully_shard
+from torch.testing._internal.common_cuda import TEST_CUDA
+from torch.testing._internal.common_fsdp import FSDPTestMultiThread
```

Comment:
I personally wanted to keep it separate for organization and moving faster, but it is not a dealbreaker either way.
- Organization: The dependency is unidirectional -- per-parameter FSDP tests may use utils from existing FSDP but not vice versa. Keeping separate may help avoid bloating the existing FSDP common file with code that existing FSDP tests never use.
- Moving faster: In case people change the existing FSDP common file (unlikely but possible), I would prefer not to have to manage rebase conflicts through this stack of PRs 😃 .

Reply:
sounds good yeah!